themes/_base.sass

```diff
@@ -0,0 +1,11 @@
+@mixin component($component)
+  .ais-#{$component}
+    @content
+
+@mixin modifier($modifier)
+  &--#{$modifier}
+    @content
+
+@mixin element($element)
+  &__#{$element}
+    @content
```

Comment:
Hmmm, the good thing it that is doesn't nest anything :) The nesting is just use to build the class names from the parent using that `&--` and `&__` syntax.